src/altair/helpers.rs

```diff
@@ -44,9 +50,37 @@ pub fn get_next_sync_committee_indices<
         MAX_VALIDATORS_PER_COMMITTEE,
         SYNC_COMMITTEE_SIZE,
     >,
-    _context: &Context,
-) -> Result<()> {
-    Ok(())
+    context: &Context,
+) -> Result<Vec<ValidatorIndex>> {
+    // Return the sync committee indices, with possible duplicates, for the next sync committee.
+    let epoch = get_current_epoch(state, context) + 1;
+    let max_random_byte = u8::MAX as u64;
+    let active_validator_indices = get_active_validator_indices(state, epoch);
+    let active_validator_count = active_validator_indices.len();
+    let seed = get_seed(state, epoch, DomainType::SyncCommittee, context);
+    let i = 0;
+    let mut sync_committee_indices = Vec::<ValidatorIndex>::new();
```

Comment:
we could do `Vec::with_capacity` as we know the length of this thing ahead of time